src/Microsoft.DotNet.XHarness.CLI/CommandArguments/WASM/WasmTestBrowserCommandArguments.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Mono.Options;
+
+namespace Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm
+{
+    /// <summary>
+    /// Specifies a name of a JavaScript engine used to run WASM application.
+    /// </summary>
+    internal enum Browser
```

Comment:
This enum is probably not needed

Reply:
Given we'll likely soon extend it with `Firefox`, `Safari` etc I think it's fine to keep this.